deployment/deploy-presidio.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+REGISTRY=${1:-mcr.microsoft.com}
+TAG=${2:-latest}
+helm install presidio-demo --set registry=$REGISTRY,tag=$TAG ../charts/presidio --namespace presidio
```

Comment:
presidio-demo should be an argument, and default should be "demo" (see other comment)